### PR TITLE
V0.9.3

### PR DESCRIPTION
**Pull request by caoxiemeihao**
The `v0.9.3` for support Worker [vite-plugin-electron/issues/69](https://github.com/electron-vite/vite-plugin-electron/issues/69)

image

### CODE REVIEW

**Comment by caoxiemeihao** [comment]
@astoilkov 👋
Not sure if the throw message is accurate. 🤔

File: packages/electron-renderer/src/use-node.js.ts [hunk 1]
```diff
@@ -88,31 +88,57 @@ const electron = `
...
+    'rawListeners',
+    'removeAllListeners',
+    'removeListener',
+    'setMaxListeners',
+  ];
+  for (const key of keys) {
+    _ipcRenderer[key] = () => {
+      throw new Error(
+        '"ipcRenderer" does not work in "Worker"\\n' +
+        'You can see https://github.com/electron-vite/vite-plugin-electron/issues/69'
```
Comment (caoxiemeihao) [inline packages/electron-renderer/src/use-node.js.ts:124]: @astoilkov 👋
Not sure if the throw message is accurate. 🤔

File: packages/electron-renderer/src/use-node.js.ts [hunk 2]
```diff
@@ -88,31 +88,57 @@ const electron = `
...
+      );
```
Comment (caoxiemeihao) [inline packages/electron-renderer/src/use-node.js.ts:125]: @astoilkov 👋
Not sure if the throw message is accurate. 🤔

Comment (astoilkov) [reply packages/electron-renderer/src/use-node.js.ts:125]: I would update the first sentence like that: "ipcRenderer does not work in a Web Worker". The second one is good.

Comment (caoxiemeihao) [reply packages/electron-renderer/src/use-node.js.ts:125]: Can you a submit PR to v0.9.3 branch 😅

**Comment by PAXANDDOS** [comment]
This looks just great! 🚀 

I'm sorry I couldn't take a look, I was busy upgrading my laptop the whole weekend, and also reinstalled and reconfigured my OS. Ordered myself a brand new Samsung SSD 980 500GB and everything is blazingly fast now! Also upgraded RAM to Kingston Fury 16GB DDR4 🚀

**Comment by caoxiemeihao** [comment]
Looks like a nice hardware upgrade! 🚀

**Pull request merged**